codenames/codenames.proto

```diff
@@ -0,0 +1,77 @@
+syntax = "proto3";
+
+package Codenames;
+
+message StartGameSettings {
+    bool isLessThan4Users = 1;
```

Comment:
Couldn't you get this by counting the user?...

Reply:
Could. But if user is less than 4 the rules of the game changes. And I haven't decide when to implement it, just leave it be a setting.